core-plugins/src/main/java/io/cdap/plugin/batch/aggregator/GroupByAggregator.java

```diff
@@ -107,24 +108,31 @@ public class GroupByAggregator extends RecordReducibleAggregator<AggregateResult
     put("ANYIF", "AnyIf");
   }};
 
+  // Ansi SQL aggregations
   private final HashMap<GroupByConfig.Function, String> functionSqlMap =
     new HashMap<GroupByConfig.Function, String>() {{
-    put(GroupByConfig.Function.AVG, "AVG(%s)");
-    put(GroupByConfig.Function.COUNT, "COUNT(%s)");
-    put(GroupByConfig.Function.MAX, "MAX(%s)");
-    put(GroupByConfig.Function.MIN, "MIN(%s)");
-    put(GroupByConfig.Function.STDDEV, "STDDEV_POP(%s)");
-    put(GroupByConfig.Function.SUM, "SUM(%s)");
-    put(GroupByConfig.Function.VARIANCE, "VAR_POP(%s)");
-    put(GroupByConfig.Function.COLLECTLIST, "ARRAY_AGG(%s IGNORE NULLS)");
-    put(GroupByConfig.Function.COLLECTSET, "ARRAY_AGG(DISTINCT %s IGNORE NULLS)");
-    put(GroupByConfig.Function.COUNTDISTINCT, "COUNT(DISTINCT %s) + IF(COUNTIF(%<s IS NULL) > 0, 1, 0)");
-    put(GroupByConfig.Function.COUNTNULLS, "COUNTIF(%s IS NULL)");
-    put(GroupByConfig.Function.CONCAT, "STRING_AGG(CAST(%s AS STRING), \", \")");
-    put(GroupByConfig.Function.CONCATDISTINCT, "STRING_AGG(DISTINCT CAST(%s AS STRING) , \", \")");
-    put(GroupByConfig.Function.LOGICALAND, "COALESCE(LOGICAL_AND(%s), TRUE)");
-    put(GroupByConfig.Function.LOGICALOR, "COALESCE(LOGICAL_OR(%s), FALSE)");
-  }};
+      put(GroupByConfig.Function.AVG, "AVG(%s)");
+      put(GroupByConfig.Function.MAX, "MAX(%s)");
+      put(GroupByConfig.Function.MIN, "MIN(%s)");
+      put(GroupByConfig.Function.STDDEV, "STDDEV_POP(%s)");
```

Comment:
This looks BQ-specific, also VAR_POP

Reply:
Per https://www.oreilly.com/library/view/sql-in-a/9780596155322/ch04s02.html the VAR_POP and STDDEV_POP functions are ANSE SQL standard

Reply:
Every day you learn something new!